src/ints/mouse.cpp

```diff
@@ -763,63 +764,75 @@ static Bitu INT33_Handler(void) {
 		else if ((int16_t)reg_dx != POS_Y) mouse.y = static_cast<float>(reg_dx);
 		DrawCursor();
 		break;
-	case 0x05:	/* Return Button Press Data */
+	case 0x05: // MS MOUSE v1.0+ / CuteMouse - return button press data / mouse wheel data
 		{
-			uint16_t but=reg_bx;
-			reg_ax=mouse.buttons;
-			if (but>=MOUSE_BUTTONS) but = MOUSE_BUTTONS - 1;
-			reg_cx=mouse.last_pressed_x[but];
-			reg_dx=mouse.last_pressed_y[but];
-			reg_bx=mouse.times_pressed[but];
-			mouse.times_pressed[but]=0;
+            uint16_t but = reg_bx;
+            if (but == 0xffff && mouse.cute_mouse) {
```

Comment:
What's `0xffff`? Some explanation or constant extraction would be helpful here.

Reply:
Added some explanation